Cargo.toml

```diff
@@ -77,6 +77,6 @@ go = ['dep:wit-bindgen-go']
 
 [dev-dependencies]
 heck = { workspace = true }
-wasmtime = { version = "10", features = ['component-model'] }
+wasmtime = { git = "https://github.com/dicej/wasmtime", branch = "resources", features = ["component-model"] } #{ version = "10", features = ['component-model'] }
```

Comment:
as master has progressed to version 11 which includes Joel's changes, this is no longer necessary (fixing a source of conflict)